applications/cup2024-robot-motors/include/real/app_conf.hpp

```diff
@@ -20,3 +55,182 @@ constexpr float motor_lift_speed_pid_integral_limit = 500 / motor_lift_speed_pid
+    lift_limits::min_speed_mm_per_period,
+    lift_limits::max_speed_mm_per_period,
+    lift_limits::max_acceleration_mm_per_period2,
+    false,
+    lift_anti_blocking::speed_threshold,
+    lift_anti_blocking::error_threshold,
+    lift_anti_blocking::blocked_cycles_threshold
+    );
+
+/// @brief Motor Lift SpeedPIDControllerParameters.
+static cogip::motion_control::SpeedPIDControllerParameters motor_lift_speed_pid_parameters(&motor_lift_speed_pid);
+
+enum {
+    MOTOR_LIFT_ID = static_cast<uint8_t>(cogip::actuators::Enum::MOTOR_LIFT)
+};
+
+/// @brief Lifts Motor driver
+static cogip::motor::MotorDriverDRV8873 lifts_motor_driver(actuators_motors_params);
+
+/// @brief Motors
+static cogip::motor::MotorRIOT lift_motor(lifts_motor_driver, MOTOR_LIFT_ID);
+
+/// @brief Lift motor encoder
+static cogip::encoder::EncoderQDEC lift_motor_encoder(MOTOR_LIFT_ID,
+    cogip::encoder::EncoderMode::ENCODER_MODE_X4,
+    lift_motor_configuration::wheels_encoder_resolution);
+
+static cogip::localization::OdometerEncoderParameters lift_motor_odometer_params{
+    /* pulse_per_mm     */ lift_motor_configuration::pulse_per_mm,
+    /* reverse_polarity */ true
+};
+
+/// @brief Lift motor odometer
+static cogip::localization::OdometerEncoder lift_motor_odometer(lift_motor_odometer_params, lift_motor_encoder);
+
+// 1) Prepare generic motor parameters for the front lift actuator
+static const cogip::actuators::positional_actuators::MotorParameters lift_motor_params{
+    /* id                           */ cogip::actuators::Enum::MOTOR_LIFT,
+    /* default_timeout_ms           */ 2000u,
+    /* send_state_cb                */ cogip::pf::actuators::positional_actuators::send_state,
+    /* clear_overload_pin           */ CLEAR_OVERLOAD_PIN,
+    /* pose_controller_params       */ &motor_lift_pose_pid_parameters,
+    /* speed_controller_params      */ &motor_lift_speed_pid_parameters,
+    /* pose_filter_params           */ &motor_lift_pose_filter_parameters,
+    /* speed_filter_params          */ &motor_lift_speed_filter_parameters,
+    /* engine_thread_timeout_ms_    */ lift_control::control_period_ms,
+    /* motor                        */ lift_motor,
+    /* odometer                     */ lift_motor_odometer
+};
+
+// 2) Specify front-lift-specific speeds (up/down)
+static const cogip::actuators::positional_actuators::LiftParameters lift_params{
+    /* motor_params             */ lift_motor_params,       ///< Base motor parameters
+    /* init_speed_percentage    */ 100
+                                   * lift_limits::max_init_speed_mm_per_period
+                                   / lift_limits::max_speed_mm_per_period,
+                                                            ///< Init sequence speed
+    /* lower_limit              */ 0,                       ///< Lower lift limit
+    /* upper_limit              */ 160,                     ///< Upper lift limit
```

Comment:
You can precise the units for both lower and upper limits

Reply:
Done.